cmd/root.go

```diff
@@ -148,6 +148,8 @@ any client SSL certificates.`,
 		`Enables Unix sockets for all listeners using the provided directory.`)
 	cmd.PersistentFlags().BoolVarP(&c.conf.IAMAuthN, "auto-iam-authn", "i", false,
 		"Enables Automatic IAM Authentication for all instances")
+	cmd.PersistentFlags().BoolVarP(&c.conf.PrivateIP, "private-ip", "", false,
```

Comment:
`BoolVarP` means you'll provide a long flag and a short flag. We should decide if private-ip warrants a short flag. I think we could skip the short flag for now and just use `BoolVar`